src/test/rustdoc-ui/intra-doc/feature-gate-pointer.stderr

```diff
@@ -0,0 +1,11 @@
+error[E0658]: rustdoc does not allow disambiguating `*const` and `*mut`, and pointers/references are unstable until it does
+  --> $DIR/feature-gate-pointer.rs:1:1
+   |
+LL | //! [pointer::add]
+   | ^^^^^^^^^^^^^^^^^^
```

Comment:
This span doesn't look right. Can you use the same span that e.g. `broken_intra_doc_links` uses?

Reply:
Huh, I didn't even realize it was different. Good catch, thanks!